### PR TITLE
Add fetch period day page insights besides original period week data

### PR DESCRIPTION
**Pull request by grimmerk**
## Types of changes

- **New feature**

## Checklist:
- [ ] Add test cases to all the changes you introduce
- [ ] Run `poetry run pytest` locally to ensure all linter checks pass
- [ ] Update the documentation if necessary

### CODE REVIEW

File: dags/ods/fb_page_insights/udfs/fb_page_insights_data_uploader.py [hunk 1]
```diff
@@ -129,10 +130,19 @@ def write_data_to_bigquery(
 
 def download_fb_page_insight_data_upload_to_bigquery():
     fb = FBPageInsight()
-    page_insight: PageWebInsightData = fb.get_page_default_web_insight()
-
     client = init_bigquery_client()
 
+    page_insight: PageWebInsightData = fb.get_page_default_web_insight()
```
Comment (david30907d) [inline dags/ods/fb_page_insights/udfs/fb_page_insights_data_uploader.py:135]: forgive my ignorance, why we use `get_page_default_web_insight` here but using `get_page_default_web_insight(period=Period.day)` at Line 143?

and they are all written into the same table ? 😂

Comment (grimmer0125) [reply dags/ods/fb_page_insights/udfs/fb_page_insights_data_uploader.py:135]: The first one is to query period=Period.week (default) and this one is for Period.day. Thare are period and end_time in the same table. For example, end_time=(2021,8,30) and period='week" means this data is  accumulated from 
```
start =(2021,8.30) - week = (2021, 8,23)
end  = (2021,8,30)
```
In FB query design, you can specify end_time and period.

Comment (tai271828) [reply dags/ods/fb_page_insights/udfs/fb_page_insights_data_uploader.py:135]: Thanks for the explanation!

**Review by tai271828 — APPROVED** [review]

**Comment by grimmer0125** [comment]
@tai271828 , @david30907d  I'm going to merge this PR.

**Pull request merged**